extension/src/accessors.rs

```diff
@@ -545,4 +545,26 @@ pub mod toolkit_experimental {
             }
         }
     }
+
+    pg_type! {
+            #[derive(Debug)]
+            struct AccessorPercentileArray<'input> {
+            len: u64,
+            percentile: [f64; self.len],
+            }
+    }
```

Comment:
This seems to be formatted differently than other accessor structs. Granted rustfmt doesn't seem to mind (although I'm surprised it doesn't)

Reply:
Not sure why rustfmt wasn't fixing that initially but I think I got it updated correctly now.